core/templates/dev/head/domain/utilities/UrlInterpolationService.js

```diff
@@ -67,7 +67,7 @@ oppia.factory('UrlInterpolationService', ['alertsService',
 
       // Parameter values can only contain alphanumerical characters, spaces,
```

Comment:
I would very much prefer not implementing this change. I struggled trying to find whether ``$http.get()`` supports this. Angular's documentation is really, really bad. I finally found other code in Oppia showing this behavior (the internet also did not seem to know when I searched before). Thanks for pointing this out.